smithy-openapi/src/main/java/software/amazon/smithy/openapi/model/MediaTypeObject.java

```diff
@@ -104,18 +104,18 @@ public Builder schema(Schema schema) {
             return this;
         }
 
-        public Builder example(Node example) {
+        public Builder example(ExampleObject example) {
```

Comment:
Same comment about leaving the Node variants of these builder methods.

Reply:
resolved.